src/BenchmarksDriver/Program.cs

```diff
@@ -21,40 +21,54 @@ public class Program
 
         private static readonly HttpClient _httpClient = new HttpClient();
 
+        private static readonly IEnumerable<string> _commonHeaders = new string[]
+        {
+            "Host: localhost",
+            "Accept: {0}",
+            "Connection: keep-alive"
+        };
+
+        private static readonly IEnumerable<string> _plaintextHeaders =
```

Comment:
These look pretty awkward. Why not just include Host and Connection headers in _commonHeaders and make _plaintextHeaders = _commonHeaders.Append(...)